website/public/js/controllers/menuCtrl.js

```diff
@@ -43,5 +43,13 @@ angular.module('habitrpg')
       $scope.hasNoNotifications = function() {
         return selectNotificationValue(false, false, false, false, true);
       }
+
+      $scope.seenRemovedFromGroupNotification = function() {
```

Comment:
If the user is kicked from multiple groups, will this remove all notifications about it after the user has opened the modal for one of them?